src/util.rs

```diff
@@ -128,3 +132,35 @@ pub fn output_report(report: &CrashReport, matches: &ArgMatches, argv: &[&str])
     }
     Ok(())
 }
+
+pub fn change_ignored_frames(path: &Path) -> error::Result<()> {
+    let file = std::fs::File::open(path)
+        .with_context(|| format!("Cannot open file: {}", path.display()))?;
+    let mut reader = BufReader::new(file)
+        .lines()
+        .map(|x| x.unwrap())
+        .collect::<Vec<String>>();
+    if reader.is_empty() || !reader[0].contains("FUNCTIONS") && !reader[0].contains("FILES") {
+        return Err(error::Error::Casr(format!(
+            "File {} is empty or does not contain \
+                    FUNCTION or FILES on the first line",
+            path.display()
+        )));
+    }
+    let (funcs, paths) = if reader[0].contains("FUNCTIONS") {
+        if let Some(bound) = reader.iter().position(|x| x.contains("FILES")) {
+            let files = reader.split_off(bound);
+            (reader, files)
```

Comment:
We shouldn't replace filters with the user specified once. Instead we should append user filters to default filters.

Reply:
done